src/Loader.php

```diff
@@ -329,7 +315,7 @@ public function __construct(
 
         /** Calculate and build various paths. */
         foreach (['CachePath', 'QuarantinePath', 'SignaturesPath'] as $Path) {
-            if (!$$Path) {
+            if (!${$Path}) {
```

Comment:
Not sure whether we need that change. Still executes fine either way.

Reply:
It is more readable i think